modules/core/src/main/java/io/fluo/core/impl/TransactorCache.java

```diff
@@ -70,7 +70,10 @@ private void logTimedoutTransactor(Long transactorId, long lockTs, Long startTim
         lockTs);
   }
 
-  public void addTimedoutTransactor(final Long transactorId, final long lockTs, final Long startTime) {
+  // @formatter:off
```

Comment:
I turned off formatting as the formatter was putting the method on one line that is > 100 chars which cause checkstyle to fail.  This happens often with string text blocks but you can just break them up.  For some reason (not sure), a few methods/statements would go back to >100 char lines if the formatter was re-run.  Therefore, I just turned off formatting at these locations.  

In 4a7ebabe4b672, I added comments as to why formatting is turned off in each location.